diff --git a/gcc/graphite-poly.h b/gcc/graphite-poly.h
index aa7b07c..6a7d1e2 100644
--- a/gcc/graphite-poly.h
+++ b/gcc/graphite-poly.h
@@ -374,7 +374,7 @@ struct poly_bb
    out dimension of transformed to count the scatterting transform dimension.
    */
 #define PBB_NB_LOCAL_VARIABLES(PBB) (0)
-#define PBB_NB_SCATTERING_TRANSFORM(PBB) (isl_map_n_out (PBB->transformed))
+#define PBB_NB_SCATTERING_TRANSFORM(PBB) (isl_map_dim (PBB->transformed, isl_dim_out))
 #define PBB_IS_REDUCTION(PBB) (PBB->is_reduction)
 
 extern poly_bb_p new_poly_bb (scop_p, void *);
